extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/client/runtime/OidcClientsImpl.java

```diff
@@ -38,9 +39,20 @@ public OidcClient getClient(String id) {
 
     @Override
     public void close() throws IOException {
-        defaultClient.close();
```

Comment:
@fwippe what about the default client ? Do you mean it can be closed twice if both `OidcClients` and `OidcClient` is injected ? The same can happen then with the named clients.
I propose to keep it simple - simply catch `Exception` whatever it is and ignore it - we don't really want to propagate some low level IO related issue to do with double closing it, etc

Reply:
Or perhaps  better - add `closed` flag and throw ISE exception in its methods if they are invoked on the closed client - except for close() itself

Reply:
@fwippe In fact I think we should have it addressed in a dedicated PR - so that we can backport it - I'd revert all `close` related changes to `OidcClients`